templates/core/core-dpl.yaml

```diff
@@ -86,7 +86,14 @@ spec:
               secretKeyRef:
                 name: "{{ template "harbor.jobservice" . }}"
                 key: JOBSERVICE_SECRET
-         {{- if .Values.internalTLS.enabled }}
+          {{- if .Values.existingSecretAdminPassword }}
```

Comment:
why not simply add another if-guarded line to the secret? This kind of breaks the convention here.

Reply:
Maybe I don't catch your comment.

But if the comment is to set it in "core" secret that is generated, I guess we can't
The goal here is to retrieve the admin password value from a "previously generated" one.
Then, we cannot add it to the core secret.
And then we need to set it this way AFAIK.

please don't hesitate to correct me as I may didn't understood your comment

Reply:
Absolutely right. My fault.